package.json

```diff
@@ -23,17 +23,21 @@
     "chalk": "^2.4.2",
     "chokidar": "^3.0.1",
     "content-disposition": "^0.5.3",
+    "cors": "^2.8.5",
     "debounce": "^1.2.0",
+    "express": "^4.18.2",
     "extract-zip": "^1.6.7",
     "findup-sync": "^3.0.0",
     "fs-extra": "^8.1.0",
     "ignore": "^5.1.4",
     "jest": "^26.6.3",
     "js-yaml": "^4.1.0",
+    "mkcert-cli": "^1.5.0",
     "moment": "^2.24.0",
+    "node-fetch-commonjs": "^3.3.2",
```

Comment:
Just want to double check if we need this or if we can make use of the existing `http` module